IPs/RSKIP60.md

```diff
@@ -0,0 +1,89 @@
+# Checksum Address Encoding
+```
+RSKIP:
+	Title: Checksum Address Encoding
+	Authors:
+		Julian Len <julianlen@gmail.com>
+		Ilan Olkies <ilanolkies@outlook.com>
+	Type: Standard Track
+	Created: 2018-25-06
+```
+## Motivation
+
+- Avoid typing confusion in adresses.
+- Differentiate addresses of different networks.
+
+## Abstract
+
+Addresses can be validated using an injective function that makes capital letters redundant.
+RSKIP-0060 **describes an address checksum mechanism** that can be implemented in any network based on [EIP-55](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-55.md).
+
+## Specification
+```python
+def checksum_encode(addr, chain_id = None):
+    o = '0x'
+    addr_prefix = '' if None else str(chain_id) + addr.lower()
```

Comment:
This line is wrong, if addr is `b''` then it makes no sense to do `.lower` and if it is a string then it makes no sense to to `.hex()` in the following lines. `if None` is constant.